please review code

jay-rajput30 opened the pull request.

jay-rajput30 committed:
initial commit

shubham-singh commented on index.html:
```diff
@@ -0,0 +1,89 @@
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
```
Add a favicon image

shubham-singh commented on pages/components.html:
```diff
@@ -0,0 +1,451 @@
+          id="doc--description--buttons"
+          class="doc--description--buttons"
+        >
+          <button class="button primary--button">primary</button>
```
Try giving 
cursor: pointer

to all buttons. It give an important visual feedback to the user that it is interactive

shubham-singh reviewed (commented):
The navigation font size looks small on desktop and too condensed. Rest all looks good.

grvsharma1810 commented on app.js:
```diff
@@ -0,0 +1,26 @@
+
+toggleBtn.addEventListener("click", function(){
+    toggleBtn.classList.toggle("show--hide--rotate");
+    aside.classList.toggle("show");
```
I didn't knew classList has a toggle method too. This is looks nice.

grvsharma1810 commented on app.js:
```diff
@@ -0,0 +1,26 @@
+//         description.classList.add("description-translate");
+//     })
+// })
+
```
Please remove commented code.

ashrayparmar replied:
you can delete unused code

grvsharma1810 commented on app.js:
```diff
@@ -0,0 +1,26 @@
+    toggleBtn.classList.toggle("show--hide--rotate");
+    aside.classList.toggle("show");
+    // description.classList.toggle("description--translate");
+
```
Please remove commented code.

grvsharma1810 commented on app.js:
```diff
@@ -0,0 +1,26 @@
+// aside.addEventListener("mouseenter", function(){
+//     aside.classList.add("show");
+
+// })
```
Please remove commented code.

grvsharma1810 commented on index.html:
```diff
@@ -0,0 +1,89 @@
+        <li class="doc--nav--items"><a href="./pages/components.html">docs</a></li>
+        <li class="doc--nav--items"><a href="#connect">connect</a></li>
+      </ul>
+    </nav> -->
```
This too.

grvsharma1810 commented on style.css:
```diff
@@ -0,0 +1,356 @@
+  background-attachment: fixed;
+  background-size: cover; */
+}
+
```
remove this.

grvsharma1810 commented on style.css:
```diff
@@ -0,0 +1,356 @@
+}
+.primary--button:active{
+  transform: scale(0.96);
+} */
```
remove this.

grvsharma1810 commented on style.css:
```diff
@@ -0,0 +1,356 @@
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
```
remove this.

grvsharma1810 reviewed (commented):
Hey Jay, I have some minor suggestions for doc site - 
1. Please reduce the padding in navbar. It looks very odd.
2. In desktop view, fix sidebar and don't hide it but in mobile view, show and hide sidebar on button click as you are doing now.
3. Add "cursor : pointer" to all the buttons as Shubham said. 

These are some minor suggestions. Other than this code is well written and clean.

ashrayparmar commented on index.html:
```diff
@@ -0,0 +1,89 @@
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
```
As you are using font awesome icon, you can also include it in you doc site, such that user also imports it!

ashrayparmar commented on index.html:
```diff
@@ -0,0 +1,89 @@
+        <li class="nav--items"><a href="#home">home</a></li>
+        <li class="nav--items"><a href="/pages/components.html">docs</a></li>
+        <li class="nav--items"><a href="#connect">about</a></li>
+      </ul>
```
in home and about <a> anchor tags you have not given any href links.

ashrayparmar commented on index.html:
```diff
@@ -0,0 +1,89 @@
+
+    <!-- header  section -->
+    <header id="home" class="doc--section home">
+      <h1>
```
It would be great if you just add one '-' in class name i.e class="doc-section-home".
Just a personal suggestion though.

ashrayparmar commented on style.css:
```diff
@@ -0,0 +1,356 @@
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100%25' height='100%25' viewBox='0 0 1600 800'%3E%3Cg %3E%3Cpath fill='%23484ece' d='M486 705.8c-109.3-21.8-223.4-32.2-335.3-19.4C99.5 692.1 49 703 0 719.8V800h843.8c-115.9-33.2-230.8-68.1-347.6-92.2C492.8 707.1 489.4 706.5 486 705.8z'/%3E%3Cpath fill='%234a62d3' d='M1600 0H0v719.8c49-16.8 99.5-27.8 150.7 ...
+  background-attachment: fixed;
+  background-size: cover; */
+}
```
you might want to deleted commented code

ashrayparmar commented on style.css:
```diff
@@ -0,0 +1,356 @@
+#home h1{
+  margin: 7rem var(--space-2) 5rem var(--space-2);
+  padding: var(--space-3);
+}
```
it's a better approach to go for classes instead of ID

ashrayparmar commented on index.html:
```diff
@@ -0,0 +1,89 @@
+            target="blank"
+            href="https://in.linkedin.com/in/jay-rajput-23304bab"
+            ><i class="fa fa-linkedin"></i
+          ></a>
```
as you're using { fa fa-github } icons from font awesome, please ask user to include in there project folder otherwise it will break

ashrayparmar commented on style.css:
```diff
@@ -0,0 +1,356 @@
+h5,
+h6 {
+  font-family: "Montserrat", "Courier New", Courier, monospace;
+}
```
instead of including all h1,h2,h3 .. you can just give the css property to body itself

jay-rajput30 replied:
I gave so because I wanted a different font style for heading and a different style for body

ashrayparmar reviewed (commented):
Hey Jay,
I have reviewed you PR, Just a small suggestion in the nav-bar you can have some padding in between.
Rest is all good :)

jay-rajput30 commented:
Hey guys- I have made the changes as suggested:
1. fixed the sidebar for desktop. for mobile view there is same toggle button.
2. removed commented code.
3. added cursor to buttons. 
4.  increased font-size of navbar items and adjusted navbar width

let me know if PR is good to merge.

shubham-singh commented:
Looks good, Jay!

jay-rajput30 committed:
feat: updated styles and refactored code
